drift/lib/src/runtime/query_builder/statements/query.dart

```diff
@@ -250,28 +250,29 @@ abstract class Selectable<T>
   ///
   /// Each entry emitted by this [Selectable] will be transformed by the
   /// [mapper] and then emitted to the selectable returned.
-  Selectable<N> map<N>(N Function(T) mapper) {
+  Selectable<N> map<N>(FutureOr<N> Function(T) mapper) {
     return _MappedSelectable<T, N>(this, mapper);
   }
 }
 
 class _MappedSelectable<S, T> extends Selectable<T> {
   final Selectable<S> _source;
-  final T Function(S) _mapper;
+  final FutureOr<T> Function(S) _mapper;
 
   _MappedSelectable(this._source, this._mapper);
 
   @override
-  Future<List<T>> get() {
+  Future<List<T>> get() async {
```

Comment:
It looks like this doesn't need to be `async`?

Reply:
That's right, my bad :)